jwt.d.ts

```diff
@@ -37,21 +45,6 @@ export type UserType = FastifyJWT extends { user: infer T }
   ? T
   : SignPayloadType
 
-// standard names https://www.rfc-editor.org/rfc/rfc7515.html#section-4.1
-// same interface as in jsonwebtoken lib - https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/jsonwebtoken/index.d.ts
-export interface JwtHeader {

```

Comment:
Should we export the JwtHeader?
It seems that it was not tested this export

Reply:
But I've added tests in `fast-jwt` where the `JwtHeader` is defined.

https://github.com/nearform/fast-jwt/blob/47ae343727d2bea251d62f59ae99f9dd4c39b435/test/types.spec.ts#L62-91

Reply:
I mean in the same way, we do for other packages. As an example:
https://github.com/fastify/fastify/blob/main/fastify.d.ts#L179

We (re) export the `fast-jwt` types

Reply:
Re exported in https://github.com/fastify/fastify-jwt/pull/195/commits/902c478a35ecf62e1a098e7459d72c068bf11cca